src/models/ClientSearch.php

```diff
@@ -16,6 +16,9 @@
 
 class ClientSearch extends Client
 {
+    const DEBT_LABEL_CREDITOR = 'creditor';
+    const DEBT_LABEL_DEBITOR = 'debitor';
```

Comment:
```suggestion
    const DEBT_LABEL_DEBTOR = 'debtor';
```

В английсокм языке нет слова `Debitor`